SPECS/nodejs/nodejs18.spec

```diff
@@ -1,12 +1,17 @@
 # Retrieved from 'deps/npm/package.json' inside the sources tarball.
-%define npm_version 9.5.1
+%define npm_version 9.8.1
+
+# Unfortunately our 'filesystem' package conflicts with /usr/lib/debug
+# and /usr/lib/debug/usr from any/all debuginfo packages, so I guess
```

Comment:
This comment seems incorrect. We publish multiple debuginfo packages providing `/usr/lib/debug` and `/usr/lib/debug/usr` and they install without issues with `filesystem`. Could you clarify the comment a bit?

Reply:
Also, why was disabling the debuginfo package require now? Earlier versions of `nodejs18` all shipped with their debuginfo packages.

Reply:
check the pipeline failure from the pr into main https://github.com/microsoft/CBL-Mariner/pull/6464

specifically https://dev.azure.com/mariner-org/mariner/_build/results?buildId=438053&view=results

Reply:
oh, i didn't know we could just ignore pipeline failures; i repushed without the debuginfo disabling commit

Reply:
The buddy build pipelines are a bit simplified to be able to perform a faster sanity check. The ones ran in the PR check and in any "official" builds (nightly dev build and monthly + fast-track release builds) represent the proper build environment and also run install/uninstall tests to make sure the new packages don't fail.